transitions/SwipeLeft.glsl

```diff
@@ -0,0 +1,10 @@
+// Author:CheneyChan
+// License: MIT
+
+vec4 transition (vec2 uv) {
+  if (1.0 - uv.x > progress) {
+    return getFromColor(vec2(uv.x + progress, uv.y));
+  } else {
+    return getToColor(vec2(uv.x + progress - 1.0, uv.y));
+  }
```

Comment:
this would be more efficient for glsl if it was using the mix function (mix from and to using 0 and 1)

not tested but i think something like

`return mix(getFromColor(..), getToColor(..), float(1.0 - uv.x > progress))`